static/js/services/enketo.js

```diff
@@ -24,47 +23,30 @@ angular.module('inboxServices').service('Enketo',
     var FORM_ATTACHMENT_NAME = 'xml';
     var REPORT_ATTACHMENT_NAME = this.REPORT_ATTACHMENT_NAME = 'content';
 
-    var replaceJavarosaMediaWithLoaders = function(formInternalId, form) {
```

Comment:
👍 
Can't remember why the `internalId` exists, but it certainly complicates things.  If it's genuinely required, it would be nice to force it to match the `docId` (minus the `form:` prefix).